.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 88
```

Comment:
Since we're using Black with the default 88 chars max line length, we should set up Flake8 accordingly :slightly_smiling_face: